.travis.yml

```diff
@@ -23,6 +24,7 @@ script:
   - make elixir
   - make fortran
   - make go
+  - make haskell
```

Comment:
Missing `make haskell` in Makefile. You should add command what `make haskell` do.

Example in Makefile:

... [Another command]
haskell:
        [command to run haskell and the code]
... [Another command]

Looks like you updated how to install haskell.